.cirrus.yml

```diff
@@ -731,6 +742,13 @@ podman_machine_task:
          $CIRRUS_CHANGE_TITLE !=~ '.*CI:DOCS.*' &&
          $CIRRUS_CHANGE_TITLE !=~ '.*CI:BUILD.*'
         ) || $CIRRUS_CRON == "main"
+    # skip: - when PR (we never want to skip on nightly tests)
+    #       - when no danger files are changed
+    #       - when no machine code files are changed
+    skip: &skip_machine_test >-
+        $CIRRUS_PR != '' &&
+        !changesInclude('Makefile', 'contrib/cirrus/**', 'vendor/**', 'hack/**') &&
+        !changesInclude('cmd/podman/machine/**', 'pkg/machine/**', '**/*machine*.go')
```

Comment:
Similar to my `bud` comments. Is it possible for changes to something-else.go to break podman-machine tests?

Reply:
Ugh, I meant to post these in #23030

Reply:
Well I cannot say 100% no. Sure there is risk that some day something weird happens or we update the file structure. Nightly runs should catch things and I also count on reviewers to see that machine tests are marked skipped but they changed some machine related code.